app/controllers/manifests_controller.rb

```diff
@@ -1,4 +1,9 @@
 class ManifestsController < ApplicationController
   def new
   end
+
+  def index
+    result = HTTParty.get(api_v0_manifests_search_url, headers: { "Content-Type" =>'application/json'})
```

Comment:
updated to use `Manifest.all` -- figure we will use scope once you submit your PR against this PR!